pyqtgraph/colormap.py

```diff
@@ -221,29 +239,28 @@ class ColorMap(object):
 
     def __init__(self, pos, color, mapping=CLIP, mode=None): #, names=None):
         """
-        ===============     =======================================================================
-        **Arguments:**
-        pos                 Array of positions where each color is defined
-        color               Array of colors.
-                            Values are interpreted via 
-                            :func:`mkColor() <pyqtgraph.mkColor>`.
-        mapping             Mapping mode (ColorMap.CLIP, REPEAT, MIRROR or DIVERGING)
-                            controlling mapping of relative index to color. String representations
-                            'clip', 'repeat', 'mirror' or 'diverging' are also accepted.
-                            CLIP maps colors to [0.0;1.0] and is the default.
-                            REPEAT maps colors to repeating intervals [0.0;1.0];[1.0-2.0],...
-                            MIRROR maps colors to [0.0;-1.0] and [0.0;+1.0] identically
-                            DIVERGING maps colors to [-1.0;+1.0]
-        ===============     =======================================================================
+        __init__(pos, color, mapping=ColorMap.CLIP)
+        
+        Parameters
+        ----------
+        pos: array_like of float in range 0 to 1
+            Assigned positions of specified colors
+        color: array_like of colors
+            List of colors, interpreted via 
+            :func:`mkColor() <pyqtgraph.mkColor>`.
+        mapping: str or int, optional
+            Controls
```

Comment:
get rid of newline?

Reply:
Probably. 
Sphinx has some kind of obscure method to determine line breaks in the parameter definitions, though.

With the bonus newline, this renders as 
mapping _(str or int, optional)_ - Controls [text continues]

Without the bonus newline, I _think_ this rendered as
mapping _(str or int, optional)_ -
Controls [text continues]

I find the hanging dashes ugly...
But I agree that it is probably not worth fighting sphinx over.

⇒ will remove